src/components/templates/Audience/Audience.tsx

```diff
@@ -215,12 +215,6 @@ export const Audience: React.FunctionComponent = () => {
   const rowsForSizedAuditorium = MIN_ROWS + auditoriumSize * 2;
   const columnsForSizedAuditorium = MIN_COLUMNS + auditoriumSize * 2;
 
-  const translateRow = (untranslatedRowIndex: number) =>
-    untranslatedRowIndex - Math.floor(rowsForSizedAuditorium / 2);
-
-  const translateColumn = (untranslatedColumnIndex: number) =>
-    untranslatedColumnIndex - Math.floor(columnsForSizedAuditorium / 2);
-
```

Comment:
For reasons unbeknownst to me, the compiler suddenly complained about this causing re-evaluation of the below `useMemo` at every render. But hey, it was right. So I was happy to oblige and fix.

Reply:
Can these be defined outside of the component? If not, they should probably use `useCallback`

Reply:
Ah, I see that you've moved them into the giant `useMemo` block below. That should do the trick. Though we also probably want to clean that up in future/potentially turn it into smaller chunks.